pkg/apis/serving/v1beta1/inference_service_validation.go

```diff
@@ -141,6 +146,40 @@ func validateInferenceService(isvc *InferenceService) (admission.Warnings, error
 	return allWarnings, nil
 }
 
+// Validate multi-node variables(tensor-parallel-size, pipeline-parallel-size, WorkerSpec.Size)
+func validateMultiNodeVariables(isvc *InferenceService) error {
+	if isvc.Spec.Predictor.Model != nil {
+		if envPipelineParallelSize, exists := utils.GetEnvVarValue(isvc.Spec.Predictor.Model.PredictorExtensionSpec.Container.Env, constants.PipelineParallelSizeEnvName); exists {
+			// pipelineParallelSize should be bigger than 1 (head + worker)
+			if intPipelineParallelSize, err := strconv.Atoi(envPipelineParallelSize); err == nil {
```

Comment:
where is this `err` assignment used?
It would be useful to add it in the else clause.

Reply:
Actually, this error isn't used anywhere. The error occurs only during conversion from string to int, so we only need to provide the error message for that specific case. Additionally, it is already handling it through an else condition.

Reply:
> It would be useful to add it in the else clause.

I agree with @spolti. In the `else` you provide a generic message. Maybe, augment it with the error detail in `err`.

Reply:
added `err` into log.